src/CmsBundle/Entity/User.php

```diff
@@ -56,8 +56,20 @@ class User extends FOSUser implements TwoFactorInterface
     protected $email;
 
     /**
-     * @var string
+     * @Assert\Length(
+     *      min = 8,
+     *      max = 50,
```

Comment:
Why is 50 characters the max? I'd rather say; the max is the maximum amount of characters we allow in the database, which is 255.

Reply:
Changed to 255